src/app-layout/implementation.tsx

```diff
@@ -12,3 +13,9 @@ export const AppLayoutImplementation = React.forwardRef<AppLayoutProps.Ref, AppL
     return isRefresh ? <RefreshedAppLayout ref={ref} {...props} /> : <ClassicAppLayout ref={ref} {...props} />;
   }
 );
+
+export const createWidgetizedAppLayout = createWidgetizedForwardRef<
```

Comment:
A small comment on `createWidgetizedForwardRef`. I noticed it depends on the "appLayoutWidget" global flag. Would it make sense to move that shared code to under app layout or highlight the dependency in the util name?